### PR TITLE
Add WG related files

### PR DESCRIPTION
**Pull request by adamgreig**
I've put this as a tools repository since it's not specific to any architecture but perhaps someone has a better idea?

I've also added some fairly simple tests for the `init_data` and `zero_bss` functions, just building on the host target for now. Is there any need to also build on thumb etc?

### CODE REVIEW

**Comment by jonas-schievink** [comment]
Would it be possible to co-maintain this with the cortex-m, cortex-a, risc-v and msp430 teams?

**Comment by adamgreig** [comment]
Good idea, I've updated to be owned by the various platform teams here and in bare-metal.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 > Initialization code ("crt0") written in Rust
 
+This project is developed and maintained by the [Tools team][team].
```
Comment (jonas-schievink) [inline README.md]: I suppose this also needs updating now

File: src/test.rs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    unsafe {
+        let edata = sdata.offset(data.len() as isize);
+        init_data::<u32>(sdata, edata, sidata);
+    };
+
+    assert_eq!(data, mem);
+}
+
+#[test]
+fn test_zero_bs() {
```
Comment (jonas-schievink) [inline src/test.rs]:
```suggestion
fn test_zero_bss() {
```

**Comment by adamgreig** [comment]
I'd like to hear from some other wg members on whether they like having the project be owned by multiple teams vs tools team, but I've fixed up the README and test name.

**Comment by andre-richter** [comment]
Platform teams seem the more logical choice to me.

**Comment by therealprof** [comment]
I was thinking about this, too. Seems @japaric already had it set up for the platform teams so I kept it since it made a smidge more sense to me.

**Comment by thejpster** [comment]
I think I prefer tools team but I can live with platform teams providing they promise to play nice. By that I mean I wouldn't want cortex-m to put in changes that negatively affect msp430, for example. Good CI for all the targets will help.

Maybe we can eventually get CI to produce cycle counts for initialising fixed size blocks for each supported target, to help spot regressions.

**Review by therealprof — APPROVED** [review]
LGTM, thanks!

**Pull request merged**